rotkehlchen/chain/evm/decoding/aave/common.py

```diff
@@ -123,20 +144,25 @@ def _decode_deposit(
             if (
                 (event.location_label == user or (event.location_label == on_behalf_of and user in self.eth_gateways)) and  # noqa: E501
                 event.balance.amount == amount and
-                event.event_subtype == HistoryEventSubType.NONE
+                event.event_subtype == HistoryEventSubType.NONE and
+                (self._is_aave_token_contract(queried_address=event.address) or (event.address in {ZERO_ADDRESS, *self.eth_gateways}))  # noqa: E501
```

Comment:
Thanks @OjusWiZard , I was not aware of the aave gateway contract : https://docs.aave.com/developers/periphery-contracts/wethgateway . Do we need to admit the processing of gateway address event only in the _decode_deposit function, or is it needed also in the other functions (borrow, repay, withdraw)?